share/cmake/modules/FindYamlCpp.cmake

```diff
@@ -114,7 +125,10 @@ if(NOT YAMLCPP_FOUND)
     set(YAMLCPP_VERSION ${YamlCpp_FIND_VERSION})
     set(YAMLCPP_INCLUDE_DIR "${_EXT_DIST_ROOT}/include")
     if(WIN32)
-        set(YAMLCPP_LIBRARY "${_EXT_DIST_ROOT}/lib/libyaml-cppmd.lib")
+        if(CMAKE_BUILD_TYPE STREQUAL Debug)
+            set(_YAMLCPP_LIB_SUFFIX "d")
+        endif()
+        set(YAMLCPP_LIBRARY "${_EXT_DIST_ROOT}/lib/libyaml-cppmd${_YAMLCPP_LIB_SUFFIX}.lib")
```

Comment:
You could use `${CMAKE_STATIC_LIBRARY_SUFFIX}` instead of `.lib`or `.a`.

Reply:
Fixed.